libraries/core_libs/node/include/node/node.hpp

```diff
@@ -28,6 +28,9 @@
 #include "transaction/transaction.hpp"
 #include "vote_manager/vote_manager.hpp"
 
+namespace metrics {
```

Comment:
I guess everything what is our code should be also in taraxa namespace

Reply:
@MatusKysel so what is the purpose of taraxa namespace then ? when would you use it vs when not ?